atum/src/main/scala/za/co/absa/atum/core/ControlType.scala

```diff
@@ -0,0 +1,26 @@
+package za.co.absa.atum.core
```

Comment:
Missing copyright header

Reply:
That part is not covered by tests. It could be a better idea to make a task to add more tests?

Reply:
👍 Agree